AndroidAnnotations/androidannotations-core/androidannotations/src/test/java/org/androidannotations/helper/BundleSparseArrayActivityTest.java

```diff
@@ -0,0 +1,52 @@
+/**
+ * Copyright (C) 2010-2015 eBusiness Information, Excilys Group
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not
+ * use this file except in compliance with the License. You may obtain a copy of
+ * the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed To in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations under
+ * the License.
+ */
+package org.androidannotations.helper;
+
+import java.io.IOException;
+
+import org.androidannotations.internal.AndroidAnnotationProcessor;
+import org.androidannotations.testutils.AAProcessorTestHelper;
+import org.junit.Before;
+import org.junit.Test;
+
+public class BundleSparseArrayActivityTest extends AAProcessorTestHelper {
+
+	@Before
+	public void setUp() {
+		addManifestProcessorParameter(BundleSparseArrayActivityTest.class);
+		addProcessor(AndroidAnnotationProcessor.class);
+		addManifestProcessorParameter(BundleSparseArrayActivityTest.class, "AndroidManifest.xml");
+	}
+
+	@Test
+	public void compileSuccessForBundleSparseArray() {
+		CompileResult result = compileFiles(BundleSparseArrayCompileSuccessActivity.class);
+		assertCompilationSuccessful(result);
```

Comment:
Sorry, i think i was not clear. Please add `assertGeneratedClassContains()` here to actually check the correct method call was generated.